### DISCUSSION TITLE
Support for socketcan on linux

### DISCUSSION THREAD

**Discussion by ivanbogo** [cedricp/ddt4all#1053] [Q&A]
Recent linux kernels have built-in support for some cheap usb2can devices (for ex: https://www.inno-maker.com/product/usb-can/). 
This allows the use of sockets supported by socketcan (https://www.kernel.org/doc/html/next/networking/can.html).

Was wondering if there are plans to add support for socketcan on linux?

**Comment by KarelSvo** [comment]
https://github.com/cedricp/ddt4all/blob/master/usbdevice.py
